pandas/_libs/tslibs/period.pyx

```diff
@@ -385,6 +467,24 @@ cdef int64_t absdate_from_ymd(int year, int month, int day) nogil:
 
 
 cdef int get_yq(int64_t ordinal, int freq, int *quarter, int *year):
+    """
+    Find the year and quarter of a Period with the given ordinal and frequency
+
+    Parameters
+    ----------
+    ordinal : int64_t
```

Comment:
very odd that these take pointers

Reply:
At this point these functions are translated as directly as possible from the C versions.  In future passes I'd like to make them more pythonic too, will need to be careful about perf.

Reply:
that’s fine